__tests__/app.js

```diff
@@ -34,18 +34,20 @@ describe('generator-seedrs-npm:app', () => {
         main: 'lib/index.js',
         author: 'tech@seedrs.com',
         scripts: {
-          build: 'babel src --presets babel-preset-env --out-dir lib',
+          build: 'babel src --presets @babel/env --out-dir lib',
           commit: 'git-cz',
           lint: 'eslint ./src/*',
           test: 'node scripts/test.js',
           'semantic-release': 'semantic-release --no-ci -b $BUILDKITE_JOB_ID'
         },
         devDependencies: {
+          '@babel/core': '7.1.2',
+          '@babel/cli': '7.1.2',
+          '@babel/plugin-proposal-object-rest-spread': '7.0.0',
+          '@babel/preset-env': '7.1.0',
           '@seedrs/eslint-config-seedrs-base': '^1.1.1',
-          'babel-cli': '^6.26.0',
-          'babel-core': '^6.26.3',
-          'babel-plugin-transform-object-rest-spread': '^6.26.0',
-          'babel-preset-env': '^1.7.0',
+          'babel-core': '^^7.0.0-bridge',
```

Comment:
@risheepatel I think this should only have a single pin not two

Reply:
Ah good point. Sloppy copy and paste there. 👍